test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -136,6 +136,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'usageCollection.uiCounters.debug (boolean)',
         'usageCollection.uiCounters.enabled (boolean)',
         'vis_type_vega.enableExternalUrls (boolean)',
+        'xpack.actions.email.domain_allowlist (array)',
```

Comment:
As discussed offline, I think it's perfectly acceptable to expose this config setting to the browser :+1: